AWS.md

```diff
@@ -38,7 +38,12 @@ EC2
 
 VPC
 ---
-* Ensure you have added the correct [Gateway Endpoints](https://docs.aws.amazon.com/vpc/latest/privatelink/vpce-gateway.html) for the AWS services being accessed from your private subnets to avoid incurring unnecessary networking costs.
+* Ensure you have added the correct [Gateway Endpoints](https://docs.aws.amazon.com/vpc/latest/privatelink/vpce-gateway.html) for the AWS services being accessed from your private subnets to avoid incurring unnecessary networking costs. 
```

Comment:
I would consider putting this bullet 4th in this list, above security, as a more natural order of things to consider (`high-level -> ... -> detail`)